src/test/java/com/amazon/dlic/auth/http/jwt/keybyoidc/HTTPJwtKeyByOpenIdConnectAuthenticatorTest.java

```diff
@@ -58,6 +58,23 @@ public void basicTest() {
 		Assert.assertEquals(3, creds.getAttributes().size());
 	}
 
+
+    @Test
+    public void jwksUriTest() {
```

Comment:
Can you add a negative/unhappy test scenario as well? Demonstrate the behavior is as expected with an invalid configuration? Thank you.

Reply:
No problem, I'm on it 👍